features/sessions/src/main/java/com/android254/droidconKE2020/sessions/ui/views/DaySessionsFragment.kt

```diff
@@ -44,7 +44,19 @@ class DaySessionsFragment : Fragment(R.layout.fragment_day_sessions), SessionsCl
 
     private fun observeDaySessions() {
         sessionsViewModel.sessions.observe(viewLifecycleOwner) { sessions ->
-            setUpRvSessions(sessions)
+            if (parentFragment is BookmarkedSessionsFragment){
```

Comment:
@BKinya This implementation should change once you have the filter logic done. I mean the check on which is the parent fragment.